docker_local_deploy.sh

```diff
@@ -21,10 +21,35 @@
 docker pull library/elasticsearch:2.3
 docker run -d -p 9200:9200 elasticsearch:2.3
 
+# Start up postgres container
+docker run -e POSTGRES_USER=docker_adage -e POSTGRES_PASSWORD=password123 \
+    --name docker-postgres -p 5431:5432 -d postgres
```

Comment:
Yes, we are connecting it to port 5431 in case there is already a postgres server running on port 5432 (the traditional postgres port) in whichever machine is running this code. I know this is the case for my machine, so that way I don't have to stop the local postgres server and tie this container to port 5432.

Reply:
Oh, good call. I'd say that is a very good reason!